### PR TITLE
update of _toctree.yml and spelling fixes 

### PR DESCRIPTION
**Pull request by seshu-pavan**
Hello everyone, 
This PR has two minor changes
1. The file name's spelling for definition.mdx is corrected from defination.mdx ( nothing inside the file has been changed).
2. The .yml tree has been updated as per guide and chapter content for both of our Fundamentals and Imaging chapters. 

Best,
Fundamentals and Imaging Team

bcc- @bellabf, @aman06012003

### CODE REVIEW

**Review by ratan — APPROVED** [review]

File: chapters/_toctree.yml [hunk 1]
```diff
@@ -3,6 +3,27 @@
   - local: chapter0/1
     title: Introduction
 
+- title: Unit 1 - Fundamentals
+  sections:
+  - sub-title: Chapter 1 - Fundamentals of Computer Vision
+    sections:
+    - local: chapters/en/Unit 1 - Fundamentals/Chapter 1 - Fundamentals of Computer Vision/motivation.mdx
```
Comment (MKhalusova) [inline chapters/_toctree.yml:10]: You can remove the `.mdx` part in each entry.

**Review by MKhalusova — COMMENTED** [review]
Please remove the `.mdx` extension in all of the entries. 

Also, a broader note (not just about these changes, but about the rest of the TOC as well)The `_toctree.yml` should be under `chapters/en/`. This way, for every course translation there will be its own `_toctree.yml`. At the same time, if you move the `_toctree.yml` under `chapters/en/`, the paths in each `local` will be simpler. E.g. 
`- local: Unit 1 - Fundamentals/image_and_imaging/extension-image` instead of `- local:  chapters/en/Unit 1 - Fundamentals/image_and_imaging/extension-image`

**Comment by seshu-pavan** [comment]
I've updated with the changes you mentioned @MKhalusova . Please take a look :))

**Comment by MKhalusova** [comment]
> Please remove the `.mdx` extension in all of the entries.
> 
> Also, a broader note (not just about these changes, but about the rest of the TOC as well)The `_toctree.yml` should be under `chapters/en/`. This way, for every course translation there will be its own `_toctree.yml`. At the same time, if you move the `_toctree.yml` under `chapters/en/`, the paths in each `local` will be simpler. E.g. `- local: Unit 1 - Fundamentals/image_and_imaging/extension-image` instead of `- local: chapters/en/Unit 1 - Fundamentals/image_and_imaging/extension-image`

The changes I mentioned here should be applied to the whole TOC, and the `_toctree.yml` should be moved under chapters/en. I think we also need to include @johko in this conversation since this affects the whole course.

**Comment by johko** [comment]
Hey, as @MKhalusova has way more plan of the structuring and the toctree file, I trust her on that. So can you move the file into chapters/en @seshu-pavan ? Once you have moved it there we can merge, as far as I can see

**Comment by seshu-pavan** [comment]
I moved `_toctree.yml` from its original location `chapters` to `chapters/en/` as requested. Please take a look @MKhalusova, @johko

**Comment by seshu-pavan** [comment]
I moved the `_toctree.yml` from its original location `/chapters/` into `chapters/en/`. Please take a look and lmk @MKhalusova and @johko

**Review by MKhalusova — COMMENTED** [review]
Thanks for moving the yml file. Can you also update the local paths in the rest of the TOC?

**Comment by seshu-pavan** [comment]
I hope i changed as you expected. Please take a look @MKhalusova

**Review by MKhalusova — APPROVED** [review]
Thanks, LGTM!

**Review by johko — APPROVED** [review]
Thanks for the update :hugs: will merge it now

**Pull request merged**